source/mainnet/net/resources/ar-idp-contact.rst

```diff
@@ -19,6 +19,10 @@ Identity Providers
 
 Website: https://www.digitaltrustsolutions.nl/
 
+.. note::
+
+    On Testnet, the Concordium identity provider is a genesis identity provider that exists to provide generic identities for test purposes.
```

Comment:
This is true, but we should also mention that

"On mainnet, there are special genesis identity provider 0 and anonymity revoker 1. These are the identity provider and anonymity revoker for genesis accounts, and only genesis accounts. Their public keys were generated by a special hashing construction from digits of pi, so that their secret keys are not known. These keys will not be used for anything else."

Reply:
@tschudid Can you check what I wrote and perhaps revise it with additional details if needed?

Reply:
Looks fine to me. Maybe add the following clarifications:
"On mainnet, there are special, _non-functional_, genesis identity provider 0 and anonymity revoker 1. These are the identity provider and anonymity revoker for _the foundation's_ genesis accounts, and only _those_  accounts. Their public keys were generated by a special hashing construction from digits of pi, so that their secret keys are not known _to anyone_. These keys will not be used for anything else."

Reply:
@dg-concordium will you update to add that text?